### PR TITLE
Granite Community and Cookbooks write-up

### PR DESCRIPTION
**Pull request by adampingel**

### CODE REVIEW

File: docs/exploring/granite-community-cookbooks.markdown [hunk 1]
```diff
@@ -7,4 +7,48 @@ parent: Exploring AI System Design
...
+* [Granite Code Cookbook](https://github.com/ibm-granite-community/granite-code-cookbook)
+* [Granite Legal Cookbook](https://github.com/ibm-granite-community/granite-legal-cookbook)
+* [Granite Finance Cookbook](https://github.com/ibm-granite-community/granite-finance-cookbook)
+* [Granite Time Series Cookbook](https://github.com/ibm-granite-community/granite-timeseries-cookbook)
+
+Some of the first recipes included simple use cases that demonstrate zero-shot inference with the code and time series models.
+The recipes are growing more complex to cover data preparation (including [Data Prep Kit](https://github.com/IBM/data-prep-kit)), fine-tuning, RAG, tool use, and agentic workflows.
+
+The Granite Code Cookbook includes examples of code-specific patterns, including several that utilize analysis available from [CodeLLM Devkit](https://github.com/IBM/codellm-devkit/)
+(also see a [related blog entry](https://research.ibm.com/blog/cldk-codellm-devkit)).
```
Comment (deanwampler) [inline docs/exploring/granite-community-cookbooks.markdown:33]: How about "see also this ..."?

File: docs/exploring/granite-community-cookbooks.markdown [hunk 2]
```diff
@@ -7,4 +7,48 @@ parent: Exploring AI System Design
...
+
+The Cookbook recipes – Jupyter notebooks – are licensed under
+[CC BY 4.0](https://creativecommons.org/licenses/by/4.0/deed.en).
+Code is licensed under
+[Apache 2.0](https://opensource.org/license/apache-2-0)
+and example datasets licensed under
```
Comment (deanwampler) [inline docs/exploring/granite-community-cookbooks.markdown:39]: "example datasets are licensed"

**Review by deanwampler — APPROVED** [review]
Looks good. I added to minor wording suggestions.

**Comment by adampingel** [comment]
I'll merge now -- happy to get comments from others continuously afterward.

**Pull request merged**